src/configuration/intro.md

```diff
@@ -43,23 +43,25 @@ A simpler alternative to the `e` command is accessible from the visual mode. Typ
 [EvalSpace]
 
     >  analysis
-       asm
-       scr
        asm
        bin
        cfg
+       cmd
+       dbg
        diff
        dir
-       dbg
-       cmd
-       fs
+       emu
+       esil
+       file
+       graph
        hex
        http
-       graph
        hud
-       scr
-       search
        io
+       key
+       log
+       magic
+       pdf
```

Comment:
It's `pdb` and not `pdf`.
I would also add 3 dots in here, so it will show that the list isn't full

```suggestion
       pdb
       ...
```

Reply:
sorry my bad ^^